### PR TITLE
Initialize qe isvsvn before parsing qe identity json

### PR DESCRIPTION
**Pull request by yentsanglee**
Quoting Enclave Identity V2 has additional field tcbLevels which is used to list supported Enclave TCB levels.

Issue - quoting enclave isvsvn is not passed in as a base to compare with tcbLevels in the json file. As a result, no matching tcbLevel is found and out-of-date status is returned.

Resulotion - initialize quoting enclave isvsvn before parsing qe identity info json.

### CODE REVIEW

File: common/sgx/qeidentity.c [hunk 1]
```diff
@@ -66,6 +66,7 @@ oe_result_t oe_validate_qe_identity(
     OE_TRACE_INFO(
         "*qe_identity.qe_id_info:[%s]\n",
         sgx_endorsements->items[OE_SGX_ENDORSEMENT_FIELD_QE_ID_INFO].data);
+    platform_tcb_level.isvsvn[0] = qe_report_body->isvsvn;
```
Comment (jazzybluesea) [inline common/sgx/qeidentity.c]: Change looks good.  Please add some space and add a comment, something like "Configure the platform isvsvn from the QE report"

Comment (yentsanglee) [reply common/sgx/qeidentity.c]: Fixed

---
In reply to: [376122904](https://github.com/openenclave/openenclave/pull/2510#discussion_r376122904) [](ancestors = 376122904)

**Comment by yentsanglee** [comment]
bors

**Comment by yentsanglee** [comment]
bors try

**Pull request merged**